src/CloudModelGenerator/CodeGenerator.cs

```diff
@@ -57,7 +59,8 @@ public void GenerateTypeProvider()
 
         private void SaveToFile(string content, string fileName)
         {
-            string outputPath = _outputDir + $"{fileName}.cs";
+            string suffix = string.IsNullOrWhiteSpace(_transformFilenamesText) ? ".cs" : $".{_transformFilenamesText}.cs";
```

Comment:
I think accepting C# formatting strings would make it too complex to use. You'd have to know exactly what format to use. For example, do you input it as "{0}.mytext" or "{fileName}.mytext" or "{FileName}.mytext", etc. It seems like there's too much room for error. I'm not totally against it, but if we go that route we'd need to do some logic to make it more robust.